examples/twitter-filter/README.md

```diff
@@ -0,0 +1,49 @@
+# Tweet Analyzer
+
+## Overview
+This tweet analyzer application is implemented using the APIs provided by the `http` and `kafka` packages.
+
+## Implementation
+
+![Twitter Analyzer](twitter-analyzer.png)
+
+### Mock Twitter Server
```

Comment:
Lets explain the reason for using a mock twitter server.

Reply:
Is the existing explanation not good enough? 😅

> As it is strenuous to gain access to a Twitter developer account, a mock Twitter server is used to generate the tweets. Tweets containing random integers in the range 1 and 100000 will be generated.